src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AbstractCSharpCompletionProviderTests.cs

```diff
@@ -51,13 +51,13 @@ internal override CompletionServiceWithProviders GetCompletionService(Project pr
             SourceCodeKind sourceCodeKind, bool usePreviousCharAsTrigger, bool checkForAbsence,
             int? glyph, int? matchPriority, bool? hasSuggestionItem, string displayTextSuffix,
             string displayTextPrefix, string inlineDescription = null, bool? isComplexTextEdit = null,
-            List<CompletionFilter> matchingFilters = null, CompletionItemFlags? flags = null)
+            List<CompletionFilter> matchingFilters = null, CompletionItemFlags? flags = null, bool skipSpeculation = false)
```

Comment:
? :-/

Reply:
There's an existing compiler issue with speculation inside attributes on local functions and lambdas (see filed issue from OP). One of the impacts is that `nameof` doesn't introducing extra identifiers in scope during speculation. 
So speculation portion of these IDE tests was skipped for now (tracked with follow-up issue). It's sad that this flag had to be threaded through so many places :-/